Orange/evaluation/testing.py

```diff
@@ -317,7 +317,7 @@ def split_by_model(self):
                 res.probabilities = self.probabilities[(i,), :, :]
 
             if self.models is not None:
-                res.models = self.models[:, i]
+                res.models = self.models[:, i:i+1]
```

Comment:
I believe this was a bug because it changed self.models from 2d to 1d array. Apparently nobody needed this so far.